packages/react-instantsearch-core/src/core/__tests__/createConnector.js

```diff
@@ -69,13 +70,11 @@ describe('createConnector', () => {
         }),
       });
 
-      const wrapper = shallow(<Connected {...props} />, {
-        context,
-      });
+      const wrapper = mount(<Connected {...props} contextValue={context} />);
```

Comment:
Enzyme threw an error about objects being invalid to render, which doesn't with `mount`, however it doesn't seem like I can still reproduce it since we switched to context-less connector here

Reply:
```
Invariant Violation: ReactShallowRenderer render(): Shallow rendering works only with custom components, but the provided element type was `object`.
```

(still happens with the last test)